lib/dirac.js

```diff
@@ -471,6 +471,10 @@ dirac.tx = {
       tx[ tbl ] = new Dal( client );
       tx[ tbl ].befores = dirac.dals[ tbl ].befores;
       tx[ tbl ].afters  = dirac.dals[ tbl ].afters;
+      tx[ tbl ].lock = function(mode, callback) {
```

Comment:
> I still feel that if it's a table-specifc operation, tx or not, it should be on the dal
That's true, and if users run into the error, they'll immediately know they need to wrap it in a transaction.